bench/add.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 Eideticom
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+int entry(char *input, int inp_sz) {
```

Comment:
I think these programs should probably be compiled in the makefile so the unit tests can at least be sure that they compile.